src/RunTests/build/Microsoft.Build.RunVSTest.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+	<PropertyGroup>
+		<NugetPath Condition="$(NUGET_PACKAGES)==''">$(USERPROFILE)\.nuget</NugetPath>
+		<NugetPath Condition="$(NUGET_PACKAGES)!=''">$(NUGET_PACKAGES)</NugetPath>
+		<VSTestRunnerVersion Condition="$(VSTestRunnerVersion)==''">$(MicrosoftBuildPackageVersion)</VSTestRunnerVersion>
```

Comment:
I'm not clear where `MicrosoftBuildPackageVersion` is defined

Reply:
I also am not clear how the Microsoft.Build package version related to the VSTest runner version.

Reply:
Updated this default value to be the most recent release non-preview version. However, presumably customers would specify the version they would like to use.